MineColab.ipynb

```diff
@@ -5,25 +5,14 @@
     "colab": {
       "name": "MineColab",
       "provenance": [],
-      "collapsed_sections": [],
-      "include_colab_link": true
+      "collapsed_sections": []
     },
     "kernelspec": {
       "name": "python3",
       "display_name": "Python 3"
     }
   },
   "cells": [
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "view-in-github",
-        "colab_type": "text"
-      },
-      "source": [
-        "<a href=\"https://colab.research.google.com/github/thecoder-001/MineColab/blob/master/MineColab.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
-      ]
-    },
```

Comment:
We don't want to remove this section as this allows one to directly import the notebook to their Colab from the repo. Otherwise, people would be required to either first clone the repo to their pc and then upload to Colab or input the repo's link in the new notebook dialogue box, adding to the complexity.

Reply:
I didn't remove all of that, looks like Google Colab did that by itself. I'll add the code back in a few hours (currently busy).